include/exadg/structure/spatial_discretization/operators/elasticity_operator_base.h

```diff
@@ -99,6 +99,39 @@ class ElasticityOperatorBase : public OperatorBase<dim, Number, dim>
   OperatorData<dim> const &
   get_data() const;
 
+  /*
+   * Provide near null space basis vectors used e.g. in AMG setup. deal.II allows providing constant
+   * modes as `std::vector<std::vector<double>> constant_modes_values` or
+   * `std::vector<std::vector<bool>>   constant_modes`,
+   * where `constant_modes_values` is currently checked first (within deal.II), and if not provided,
+   * `constant_modes` is used. Provide rigid body modes, override implementation in `OperatorBase`.
```

Comment:
I have only one minor comment here: I would skip the part of the documentation that is duplicated from `OperatorBase`. Things might change in dealii, so this way we might avoid that comments in ExaDG become outdated in the future.